droid-hal-discovery.spec

```diff
@@ -0,0 +1,58 @@
+# These and other macros are documented in dhd/droid-hal-device.inc
+# Feel free to cleanup this file by removing comments, once you have memorised them ;)
+
+%define device discovery
+%define vendor sony
+
+%define rpm_device discovery
```

Comment:
If you could remove this line, update dhd submodule to test with this commit https://github.com/mer-hybris/droid-hal-device/commit/0549fd71a8c3d21a6f37ef83952d00c207cbdf76 that would be great!

Reply:
I'll try later when mer obs is back online.